pkg/trafficrouting/manager.go

```diff
@@ -181,6 +180,13 @@ func (m *Manager) DoTrafficRouting(c *TrafficRoutingContext) (bool, error) {
 		if serviceModified {
 			return false, nil
 		}
+	} else if c.DisableGenerateCanaryService {
```

Comment:
plz comment the field CanaryStrategy.DisableGenerateCanaryService and explain in what case it is needed